.babelrc

```diff
@@ -1,35 +1,24 @@
 {
-  presets: [
-    [
-      "env",
-      {
-        targets: {
-          node: "current",
-          browsers: "last 2 chrome versions, last 2 firefox versions"
-        },
-        modules: false
-      }
-    ],
-    "react",
-    "flow"
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-react",
+    "@babel/preset-flow"
   ],
-  plugins: [
-    "transform-class-properties",
-    [ "transform-object-rest-spread", { "useBuiltIns": true }]
+  "plugins": [
+    ["@babel/plugin-proposal-class-properties", { loose: true }],
+    [ "@babel/plugin-proposal-object-rest-spread", { "useBuiltIns": true } ]
```

Comment:
nit: Consistent spacing. Only pointing it since my editor might rewrite it on save later.

Reply:
done